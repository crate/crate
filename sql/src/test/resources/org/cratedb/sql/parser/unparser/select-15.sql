SELECT abs(- n), ~n,
  mod(n, 3), sqrt(n),
  char_length(s), octet_length(s),
  left(s, 3), right(s, 4),
  upper(s), lower(s),
  locate(s, 'abc'), locate(s, 'xyz', 10),
  substring(s, 10), substring(s, 10, 3),
  ltrim(s), trim(trailing '!' from s),
  s||s collate en_us_ci,
  month(d), timestamp(d, '13:01:01'), timestampdiff(day, d, current_timestamp),
  next value for seq, current value for seq
FROM t
  