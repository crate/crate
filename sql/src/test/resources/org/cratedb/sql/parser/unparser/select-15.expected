SELECT (ABS(- n)), (~ n), (n MOD 3), SQRT(n), CHAR_LENGTH(s), LENGTH(s), (LEFT(s, 3)), (RIGHT(s, 4)), UPPER(s), LOWER(s), (LOCATE(s, 'abc', 1), (LOCATE(s, 'xyz', 10), (SUBSTRING(s, 10), (SUBSTRING(s, 10, 3), LTRIM(s), (TRIM(TRAILING '!' FROM s), ((s || s) COLLATE en_us_ci), MONTH(d), (TIMESTAMP(d, '13:01:01')), (TIMESTAMPDIFF(DAY, d, CURRENT_TIMESTAMP)), (NEXT VALUE FOR seq), (CURRENT VALUE FOR seq) FROM t